### PR TITLE
chore(update version)

### PR DESCRIPTION
**Pull request by juancarlostong**
## Summary
remaining versioning changes to complete https://optimizely.atlassian.net/browse/OASIS-4088

### CODE REVIEW

**Comment by juancarlostong** [comment]
do i also need to change this? https://github.com/optimizely/csharp-sdk/pull/136/files#diff-2dda2f6bba22405c672255496b3ffc55R13

**Comment by mikeng13** [comment]
nitpick: you should also change it here: https://github.com/optimizely/csharp-sdk/pull/109/files#diff-bf67f3e7c1b4668fffb73733c1c3ebfaR22

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,36 @@
...
+* Updated Pull Request template and commit message guidelines.
+### Breaking Changes:
+* `UserAttributes` objects, which are passed to API methods and returned to notification listeners, can now contain non-string values. More concretely, `UserAttributes` now extends `Dictionary<string, object>` instead of `Dictionary<string, string>`.
+### Bug Fixes:
+* Experiments and features can no longer activate when a negatively targeted attribute has a missing, null, or malformed value.
+  * Audience conditions (except for the new `exists` matcher) no longer resolve to `false` when they fail to find an legitimate value for the targeted user attribute. The result remains `null` (unknown). Therefore, an audience that negates such a condition (using the `"not"` operator) can no longer resolve to `true` unless there is an unrelated branch in the condition tree that itself resolves to `true`.
+* `SetForcedVariation` now treats an empty variation key as invalid and does not reset the variation.
+* All methods now treat an empty user ID as valid.
+* HttpClientEventDispatcher45 now logs full exception ([#112](https://github.com/optimizely/csharp-sdk/pull/112))
```
Comment (mikeng13) [inline CHANGELOG.md]: nit: wrap ` HttpClientEventDispatcher45` in ticks

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+* Datafile-version compatibility check: The SDK will remain uninitialized (i.e., will gracefully fail to activate experiments and features) if given a datafile version greater than 4.
+* Updated Pull Request template and commit message guidelines.
+### Breaking Changes:
+* `UserAttributes` objects, which are passed to API methods and returned to notification listeners, can now contain non-string values. More concretely, `UserAttributes` now extends `Dictionary<string, object>` instead of `Dictionary<string, string>`.
+### Bug Fixes:
+* Experiments and features can no longer activate when a negatively targeted attribute has a missing, null, or malformed value.
+  * Audience conditions (except for the new `exists` matcher) no longer resolve to `false` when they fail to find an legitimate value for the targeted user attribute. The result remains `null` (unknown). Therefore, an audience that negates such a condition (using the `"not"` operator) can no longer resolve to `true` unless there is an unrelated branch in the condition tree that itself resolves to `true`.
+* `SetForcedVariation` now treats an empty variation key as invalid and does not reset the variation.
```
Comment (mikeng13) [inline CHANGELOG.md]: `([#113](https://github.com/optimizely/csharp-sdk/pull/113))`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,36 @@
...
+  * `gt` (greater than) and `lt` (less than) matchers that resolve if the user has a valid number value for the targeted attribute. A valid number is a finite number in the inclusive range [-2⁵³, 2⁵³].
+  * The original (`exact`) matcher can now be used to target booleans and valid numbers, not just strings.
+* Support for A/B tests, feature tests, and feature rollouts whose audiences are combined using `"and"` and `"not"` operators, not just the `"or"` operator.
+* Datafile-version compatibility check: The SDK will remain uninitialized (i.e., will gracefully fail to activate experiments and features) if given a datafile version greater than 4.
+* Updated Pull Request template and commit message guidelines.
+### Breaking Changes:
+* `UserAttributes` objects, which are passed to API methods and returned to notification listeners, can now contain non-string values. More concretely, `UserAttributes` now extends `Dictionary<string, object>` instead of `Dictionary<string, string>`.
+### Bug Fixes:
+* Experiments and features can no longer activate when a negatively targeted attribute has a missing, null, or malformed value.
```
Comment (mikeng13) [inline CHANGELOG.md]: `([#132](https://github.com/optimizely/csharp-sdk/pull/132))`

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,36 @@
+## 3.0.0
+February 28, 2019
```
Comment (mikeng13) [inline CHANGELOG.md]: Is this realistic? Maybe put in March 1st or 4th as placeholder since we are still waiting for PRs

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,36 @@
+## 3.0.0
+March 4th, 2019
```
Comment (nchilada) [inline CHANGELOG.md]: Are we delaying the release until next week?

File: CHANGELOG.md (hunk 4 above)
Comment (nchilada) [reply CHANGELOG.md]: @mikeng13 I think I saw that you were out next week. Do we feel comfortable releasing without you? Or should we release ahead of time even if the release then precedes the documented date?

**Pull request merged**